src/bnb_trading/optimal_levels.py

```
@@ -218,213 +220,247 @@
+                "level_touches": level_touches,
+                "current_price": current_price,
+                "optimal_levels": optimal_levels,
+                "analysis_date": weekly_df.index[-1],
             }
-            
+
             logger.info("Оптимални нива намерени успешно")
             return analysis
-            
+
         except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)